### ISSUE TITLE
What is `ՐՏ_Iterable`?

### ISSUE THREAD

**Issue by Darker** [atsepkov/RapydScript#132]
For this code:

```
    def get_claim_stones(self, stone_pos, player, terminate_asap=False):
        #stone_pos = self.coord(stone_pos, -1, False)
        #print(stone_pos)
        directions = [[0, 1], [1,1], [1, 0], [1, -1], [0, -1], [-1, -1], [-1, 0], [-1, 1]]
        reverse_fields = []
        opponent_color = player.opponent_color
        for direction in directions:
            pos = [stone_pos[0]+direction[0], stone_pos[1]+direction[1]]
            #if not (pos[0]>=0 and pos[1]>=0 and pos[0]<self.rows and pos[1]<self.cols) or self.board[pos[0]][pos[1]]!=opponent_color:
            #    continue
            fields_tmp = self.find_opponent_stones(direction, pos, player)
            # It is noteworthy, that no duplicates need removal
            # because the 'beams' of search never intersect
            #reverse_fields += fields_tmp
            if terminate_asap and len(reverse_fields) >0:
                break
            #print(fields_tmp)
        #print(reverse_fields)
        #for field in reverse_fields:
        #    print("  - claims ["+str(field[0])+", "+str(field[1])+"]")
        return reverse_fields
```

I get this output:
```
var __name__ = "__main__";
function get_claim_stones(self, stone_pos, player, terminate_asap) {
    var ՐՏitr1, ՐՏidx1;
    terminate_asap = terminate_asap === void 0 ? false : terminate_asap;
    var directions, reverse_fields, opponent_color, direction, pos, fields_tmp;
    directions = [ [ 0, 1 ], [ 1, 1 ], [ 1, 0 ], [ 1, -1 ], [ 0, -1 ], [ -1, -1 ], [ -1, 0 ], [ -1, 1 ] ];
    reverse_fields = [];
    opponent_color = player.opponent_color;
    ՐՏitr1 = ՐՏ_Iterable(directions);
    for (ՐՏidx1 = 0; ՐՏidx1 < ՐՏitr1.length; ՐՏidx1++) {
        direction = ՐՏitr1[ՐՏidx1];
        pos = [ stone_pos[0] + direction[0], stone_pos[1] + direction[1] ];
        fields_tmp = self.find_opponent_stones(direction, pos, player);
        if (terminate_asap && len(reverse_fields) > 0) {
            break;
        }
    }
    return reverse_fields;
}
```
I do not understand how to make this line work: `ՐՏitr1 = ՐՏ_Iterable(directions);`

**Comment by atsepkov** [comment]
How are you running this? It should auto-generate rs_Iteraable function for you, which converts input into an array.

**Comment by Darker** [comment]
I used the online demo here: http://www.rapydscript.com/try-it/

**Comment by atsepkov** [comment]
I see, the online demo hides that function because:

- After refactoring a few months ago, I introduced a benign bug into the compiler where it includes large portions of `baselib` (a library that defines these extra methods for RapydScript) rather than just the methods being used in your code, like it meant to.
- As a result of the bug, the online compiler started dumping a lot of extra functions that the code doesn't use, cluttering the output.
- So for now I disabled showing baselib in online compiler altogether. The function will be included in your code if you compile it yourself (either by downloading from npm or even by opening developer tools on the same page and running compilation without the `omit_baselib` flag. Try the following in JavaScript console on the same page, for example: `rapydscript.compile('for i in foo: print(i)\n', { beautify: true })`.
